python/plugins/processing/CMakeLists.txt

```diff
@@ -7,8 +7,10 @@ ADD_SUBDIRECTORY(images)
 ADD_SUBDIRECTORY(modeler)
 ADD_SUBDIRECTORY(script)
 ADD_SUBDIRECTORY(tools)
-ADD_SUBDIRECTORY(tests)
 ADD_SUBDIRECTORY(ui)
+IF (ENABLE_TESTS)
```

Comment:
So you propose to keep plugin tests in their directories?
It's fine for me, just wondering about the reason.